### DISCUSSION TITLE
Keeps forgetting calendars (data loss)

### DISCUSSION THREAD

**Discussion by smu55** [bitfireAT/icsx5#82] [General]
Hi,
I have been using ICSx5 for years now and it had been working flawlessly. Thank you for your work! Recently I have regularly (about once a month) encountered the problem, that all my calendars i subscribed in ICSx5 desapear in the calendar app (aCalendar). When I go to ICSx5 the app has lost all subscribtions and I have to resubscibe them. I have not encountered a correlation with updates. My impression ist that it happens also when I do no not update. What can I do to no loose my calendars?
Thanx

**Comment by ArnyminerZ** [comment]
Currently the synchronization and data storage is made through the Android calendar provider. Maybe your device is clearing those calendars if not used for too long. What device and Android ROM/version are you using?

In any case, #76 should fix all of these issues.

**Comment by devvv4ever** [comment]
@smu55 I suspect you're running a Samsung device?

We're now hearing this more often sadly during the last year. Samsung must have a general problem when updating the devices firmware and not restoring DAVx5/ICSx5 system accounts in some rare constellations. We've not been able to reproduce this on various Samsung devices however so we can't even file a bug report there.

**Comment by smu55** [comment]
That is right! A51

21.12.2022 12:59:01 Bernhard Stockmann ***@***.***>:
